src/test/resources/sql/with/c454830c.sql

```diff
@@ -1,5 +1,5 @@
--- file:rowtypes.sql ln:314 expect:true
-with r(a,b) as
+-- file:rowtypes.sql ln:469 expect:false
+with r(a,b) as materialized
```

Comment:
This seems odd to me, that this test changed. Please make sure you're running against this version of Postgres: https://github.com/postgres/postgres/releases/tag/REL_10_6

Ideally, the default version of `psql` is also of the same version, but it's probably less important since I suspect that subsequent parsers are mostly backward compatible.

Reply:
`[ [ NOT ] MATERIALIZED ]` will be implement in version 12[1].
I run `./scripts/gen_psql_tests` with the code of actual master-branch against PostgreSQL Server 11. Therefore there is errors: 'expect' must be 'true'.
Also we could add `server_vesion` like for MySQLParser.g4[2]
Revert scripts or implement `(NOT? MATERIALIZED)?`?

I hope the parser will support 11 and higher PostgreSQL syntax. PostgreSQL 12 RC 1 released recently [3]. When would be good time to update sql sources?

---
1. https://www.postgresql.org/docs/12/sql-select.html
2.https://github.com/mysql/mysql-workbench/blob/8.0/library/parsers/grammars/MySQLParser.g4
3. https://www.postgresql.org/about/news/1975/

Reply:
Broadly speaking, I think we should focus one target at a time. Since we're close enough to supporting 10.6, I think that should be our first goal. I suspect that by getting 10.6 right, we'll be at 98%+ of accuracy for versions 11 and 12. By supporting versions a couple years old, it arguably makes this project more useful to more people because businesses don't upgrade DBs that often.

Once we get stability with this, it will be pretty easy to support versions 11 and 12. We generate the tests, see what fails, and then make the necessary tweaks. I haven't decided how supporting multiple versions in this project will work, but if you look at the official Postgres repo, you can see that they too support later versions. They do this by having branches for each (https://github.com/postgres/postgres/branches/stale). That seems sensible.

But for now, let's get something complete for 10.6.

Reply:
Good! Got it.
Now 'with' folder contains scripts from version 10.6.

I believe that maintaining of the several branches will be hard and implicit.